shared/ui-components/src/page_components/listing/listing_sidebar/ApplicationSection.tsx

```diff
@@ -3,12 +3,19 @@ import moment from "moment"
 import { Listing } from "@bloom-housing/core"
 import Apply from "@bloom-housing/ui-components/src/page_components/listing/listing_sidebar/Apply"
 import Waitlist from "@bloom-housing/ui-components/src/page_components/listing/listing_sidebar/Waitlist"
-import { openDateState } from "../../../helpers/state"
 
 export interface ApplicationSectionProps {
   listing: Listing
 }
 
+// Hide waitlist for FCFS
+const hideWaitlist = (listing: Listing) => {
+  const applicationsClosed =
```

Comment:
Ok, I think this should actually be applicationsOpen and then you can take out the ! when you evaluate it, since that will be more forgiving of null values for the waitlist data elements on the listing. The current behavior is that if waitlistMaxSize and waitlistCurrentSize are both null, it shows the box open, but my understanding (unless corrected by @slowbot) is that we shouldn't show the waitlist component in that case since there isn't one.

I guess we could also modify this to explicitly check for nulls, which is fine too but I think might be tougher to read and understand later.